README.md

```diff
@@ -1,65 +1,86 @@
+
+Streamlit lets you turn data scripts into shareable web apps in minutes, not weeks. It’s all Python, open-source, and free! And once you’ve created an app you can use our [Community Cloud platform](https://streamlit.io/cloud)  to deploy, manage, and share your app!
 
-![Example of live coding an app in Streamlit|635x380](https://raw.githubusercontent.com/streamlit/docs/main/public/images/Streamlit_overview.gif)
 
 ## Installation
 
+Open a terminal and run:
 ```bash
-pip install streamlit
-streamlit hello
+$ pip install streamlit 
+$ streamlit hello
 ```
 
-Streamlit can also be installed in a virtual environment on [Windows](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-windows), [Mac](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux), and [Linux](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux).
+This should open our sweet _Streamlit Hello_ app right in your browser.
+
+<img src="https://media.cleanshot.cloud/media/55064/9RAuEZKubvrlrlPVua50xuhyPIiD8MBdvzi7Pa3s.gif?Expires=1674790062&Signature=fqIRLW0tMXntC1vCrSq3yLPTtA58SRLiO1i0y~7YshYuOfQQokc0OI6uxWg6IBL8rw2gGnzA4ML-OdGWPgK6WSulyLR-wNbQAPiKX4LF0vXS9F6AmZ8V39Tze5Fo9TWfA3JZF3mnu7q6JrkssCk~6614xuugx9gojkOJMtsbFrOiM2kNLkje2OepBbxromYKVLLEQGwuPcFvUiA3Hb6ZuSHtNX~99wLu94PAKpXfPUf21sCt4vfmYItkl0z6BuHWYP5Zs5SlvPo8iXvdrb-dKL-~-kbTRjpcTFgWhFT1zg8iNZNCeR0Q~QvpS8rzfQPMMeIiBiRJFmTm4RC7~FRktA__&Key-Pair-Id=K269JMAT9ZF4GZ" alt="Streamlit Hello" width=500 href="none"></img>
+
+Read more on getting started with Streamlit in our [documentation](https://docs.streamlit.io/library/get-started).
 
-## A little example
+
+## API Reference
+
+#### A little example
 
 Streamlit makes it incredibly easy to build interactive apps:
 
 ```python
 import streamlit as st
-
-x = st.slider('Select a value')
-st.write(x, 'squared is', x * x)
+x = st.slider("Select a value")
+st.write(x, "squared is", x * x)
 ```
 
-<img src="https://raw.githubusercontent.com/streamlit/docs/main/public/images/simple_example.png"/>
+<img src="https://user-images.githubusercontent.com/7164864/214900507-1c89fc6b-e196-4f5c-890f-7e0be62d5d9d.png" width=300 alt="Little example"></img>
 
-## A bigger example
+#### Give me more!
 
-Streamlit's simple and focused API lets you build incredibly rich and powerful tools.  [This demo project](https://github.com/streamlit/demo-self-driving) lets you browse the entire [Udacity self-driving-car dataset](https://github.com/udacity/self-driving-car) and run inference in real-time using the [YOLO object detection net](https://pjreddie.com/darknet/yolo).
+Streamlit comes in with [a ton of additional handy elements](https://docs.streamlit.io/library/api-reference) to spice your data apps and delight your viewers. Some examples:
 
-![Final App Animation](https://raw.githubusercontent.com/streamlit/docs/main/public/images/complex_app_example.gif)
+<table border="0">
```

Comment:
really love that table! just need to make sure it also looks good on mobile (I haven't checked). Dataframes are missing though!!

Also, I would maybe change the order:

Input widgets -> Dataframes -> Chart -> Media -> Layout -> Multi-page apps -> Celebrations (or "Fun"?)

If this is getting too much, could leave out Media maybe.

Reply:
Yay, the table looks odd on mobile, I checked. Let me try with a screenshot version (instead of HTML). Might make our lives easier.

Reply:
Yep added dataframes, looks ok on mobile now that I added some CSS 
![CleanShot 2023-01-27 at 15 31 44](https://user-images.githubusercontent.com/7164864/215112155-81bfb245-5746-4ed1-9b9e-10f3662e2344.gif)